src/lib/datepicker/datepicker.md

```diff
@@ -84,38 +101,54 @@ Each validation property has a different error that can be checked:
  * A value that violates the `matDatepickerFilter` property will have a `matDatepickerFilter` error.
  
 ### Input and change events
-The input's native `input` and `change` events will only trigger due to user interaction with the
-input element; they will not fire when the user selects a date from the calendar popup. Because of
-this limitation, the datepicker input also has support for `dateInput` and `dateChange` events.
-These trigger when the user interacts with either the input or the popup.
-  
-```html
-<input [matDatepicker]="d" (dateInput)="onInput($event)" (dateChange)="onChange($event)">
-<mat-datepicker #d></mat-datepicker>
-```
+
+The input's native `(input)` and `(change)` events will only trigger due to user interaction with
+the input element; they will not fire when the user selects a date from the calendar popup. Because
+of this limitation, the datepicker input also has support for `(dateInput)` and `(dateChange)`
```

Comment:
I wouldn't use the word "limitation", since that's the expected native behavior.